enterprise/backend/src/metabase_enterprise/ai_entity_analysis/api.clj

```diff
@@ -0,0 +1,66 @@
+(ns metabase-enterprise.ai-entity-analysis.api
+  "`/api/ee/ai-entity-analysis/` routes"
+  (:require
+   [metabase-enterprise.metabot-v3.client :as metabot-client]
+   [metabase.api.macros :as api.macros]
+   [metabase.api.routes.common :refer [+auth]]
+   [metabase.premium-features.core :as premium-features]
+   [metabase.util.i18n :refer [tru]]
+   [metabase.util.malli.schema :as ms]))
+
+(set! *warn-on-reflection* true)
+
+(api.macros/defendpoint :post "/analyze-chart"
+  "Analyze a chart image using an AI vision model."
+  [_route-params
+   {model :model
+    focus-instructions :focus_instructions
+    extra-context :extra_context
+    :as _query-params}
+   {:keys [image_base64 name description timeline_events]} :- [:map
+                                                              [:image_base64 :string]
+                                                              [:name {:optional true} [:maybe :string]]
+                                                              [:description {:optional true} [:maybe :string]]
+                                                              [:timeline_events {:optional true} [:maybe [:sequential [:map
+                                                                                                                      [:name :string]
+                                                                                                                      [:description {:optional true} [:maybe :string]]
+                                                                                                                      [:timestamp :string]]]]]]]
+
```

Comment:
nit - we don't tend to add blank lines before the endpoint body ([style guide](https://guide.clojure.style/#no-blank-lines-within-def-forms))
```suggestion
```

Same goes for between forms like on line 31 although I don't mind that as much personally